terraform/modules/filecoin_node/lotus.sh

```diff
@@ -26,6 +26,10 @@ if [ -f "/root/.ssh/authorized_keys" ]; then
   chmod 0600 "/home/${NEW_USER}/.ssh/authorized_keys"
 fi
 
+#install NTP to synchronize the time differences
+timedatectl set-ntp no
```

Comment:
Why do we need to disable ntp here?

Reply:
Okay. I don't see why that is necessary, but I trust you know what you're doing.